.github/actions/build-src/action.yaml

```diff
@@ -63,8 +63,8 @@ runs:
         which python
         which pip
         pip list
-        conda info
-        conda list
+        # conda info
+        # conda list
```

Comment:
Is this assuming that `conda` is always installed on the base image?

Reply:
It should - that action is only ever used on workflows that use some kind of mamba/conda. Is it not working?

Reply:
No, it's not working. I think we are using `micromamba`, and `conda` is not there. See [failure](https://github.com/MDAnalysis/mdanalysis/actions/runs/7714658774/job/21027430191#step:6:138).

Reply:
Ah yes I remember now - on some versions of micromamba `conda` isn't aliased. Maybe switch all these to micromamba calls instead? alternatively we can just guard them with a "ignore failure" on that action step?

Reply:
Thanks for the info. I think it's good to know what's up, so I'll go with `micromamba` instead.